lib/iris/analysis/cartography.py

```diff
@@ -668,12 +668,12 @@ def project(cube, target_proj, nx=None, ny=None):
     new_cube = iris.cube.Cube(new_data)
 
     # Add new grid coords
-    x_coord = iris.coords.DimCoord(
-        target_x[0, :], 'projection_x_coordinate',
-        coord_system=copy.copy(target_cs))
-    y_coord = iris.coords.DimCoord(
-        target_y[:, 0], 'projection_y_coordinate',
-        coord_system=copy.copy(target_cs))
+    x_coord = iris.coords.DimCoord(target_x[0, :], 'projection_x_coordinate',
+                                   units='m',
```

Comment:
Essentially they _are_ physical metres. For something like Plate Carree you have to imagine a _giant_ cylindrical sheet wrapped around the planet. Points on the surface can then be projected out onto the sheet and those locations are described by the projection coordinates. Clearly a map 40000 km wide isn't particularly practical (it would be hard to lose down the back of the sofa though!) but a simple linear reduction makes things more manageable. For example a 1:110 million reduction would make the map a mere 36 cm wide.